index.html

```diff
@@ -288,6 +290,53 @@ <h5>LET'S CONNECT</h5>
 
     </section>
 
+    <section class="contact_section">
+
+        <div class="contact_form">
+
+            <header>
+                <h3>Contact me</h3>
+            </header>
+
+            <p>
+                If you have an application you are interested in developing, a feature that you need built or a project
+                that
+                needs coding. I’d love to help with it
+            </p>
+
+            <form action="/contact" method="post">
```

Comment:
- [ ] Kindly ensure that you register for an account in [formspree](https://formspree.io/html) and get a unique id. This way when users submit the form, it will be sent directly to you.